docs/contracts.rst

```diff
@@ -1072,75 +1179,106 @@ Event Log Object
 
     from web3 import Web3
     from hexbytes import HexBytes
-    w3 = Web3(Web3.EthereumTesterProvider())
+    provider = Web3.EthereumTesterProvider()
```

Comment:
Same thought as above on the `EthereumTesterProvider()`... do we just keep this simpler? This isn't a node and we don't need to tiptoe around anything by signing before sending to `eth-tester`. So should we keep the api simpler here? I can go either way but I think I lean towards the simpler setup.